tests/jax/latest/flax-wmt-wmt17_translate.libsonnet

```diff
@@ -58,9 +58,11 @@ local tpus = import 'templates/tpus.libsonnet';
     testScript+:: |||
       gsutil -q stat $(MODEL_DIR)/plugins/profile/*/*.xplane.pb
       gsutil cp -r $(MODEL_DIR)/plugins /tmp/
-      pip uninstall tensorboard_plugin_profile
-      pip install tbp-nightly
-      python3 ~/.local/lib/python3.*/site-packages/tensorboard_plugin_profile/integration_tests/tpu/tensorflow/tpu_tf2_keras_test.py --log_directory=/tmp/
+      python3 -m pip uninstall tensorboard_plugin_profile
+      python3 -m pip install tbp-nightly
+      git clone https://github.com/tensorflow/profiler.git
```

Comment:
Clive originally included integration tests as part of the pip installed outputs so that we could avoid having to git clone the assertions. The benefit of this is that there could be mismatches between the version in tbp-nightly and the version in the cloned repository which could cause the tests to fail

Reply:
fixed